app/javascript/packages/verify-flow/steps/password-confirm/personal-info-summary.spec.tsx

```diff
@@ -0,0 +1,21 @@
+import { render } from '@testing-library/react';
+import PersonalInfoSummary from './personal-info-summary';
+import type { VerifyFlowValues } from '../../verify-flow';
+
+describe('PersonalInfoSummary', () => {
+  const DEFAULT_PII: VerifyFlowValues = {
+    firstName: 'FAKEY',
+    lastName: 'MCFAKERSON',
+    address1: '1 FAKE RD',
+    city: 'GREAT FALLS',
+    state: 'MT',
+    zipcode: '59010',
+    dob: '1938-10-06',
+  };
+
+  it('renders dates accurately', () => {
```

Comment:
A better test would mock the time zone to one which would be guaranteed to change the date, since this regression test is dependent on the time zone of the computer running the test. Apparently, this is not very straight-forward to stub. I discovered [a package](https://github.com/Jimbly/timezone-mock) to do it, but it has a bug which prevents us from using it the way we need (https://github.com/Jimbly/timezone-mock/pull/58).